mkeditorpacks/LICENSE.md

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 2014, Daemon Developers
```

Comment:
That year doesn't seem right. Not that we care for this anywhere else in our various codebases, just saying. 😛